source/bdDetect.py

```diff
@@ -655,3 +688,7 @@ def addDeviceScanner(
 		scanForDevices.register(scanFunc)
 		if moveToStart:
 			scanForDevices.moveToEnd(scanFunc, last=False)
+
+
+class FallbackDevicesStore:

```

Comment:
Is there a reason why `fallBackDevices` is not a module level variable, do you expect more code to be added to `FallbackDevicesStore`?

Reply:
can you answer this please?

Reply:
see auto hidden comments from the previous review

Reply:
can you answer this please?